templates/default/sql/dml/2.6.0.sql.erb

```diff
@@ -20,4 +20,58 @@ REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("quotas_model_deploy
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_compound_columns_to_be_unique', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "row_condition": null, "condition_parser": null, "ignore_row_if": "all_values_are_missing"}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_multicolumn_sum_to_equal', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "row_condition": null, "condition_parser": null, "ignore_row_if": "all_values_are_missing"}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_select_column_values_to_be_unique_within_record', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "row_condition": null, "condition_parser": null, "ignore_row_if": "all_values_are_missing"}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_column_count_to_be_between', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "min_value": null, "max_value": null, "meta": null}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_column_count_to_equal', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "value": null, "meta": null}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_columns_to_match_ordered_list', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "row_condition": null, "condition_parser": null, "column_list": null, "column": null, "column_index": null, "meta": null}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_columns_to_match_set', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "column_set": null, "exact_match": true}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_row_count_to_be_between', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "min_value": null, "max_value": null, "meta": null}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_row_count_to_equal', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "value": null, "meta": null}');
+INSERT INTO `hopsworks`.`great_expectation` (expectation_type, kwargs_template) VALUES ('expect_table_row_count_to_equal_other_table', '{"include_config": true, "catch_exceptions": false, "result_format": "BASIC", "other_table_name": null}');
```

Comment:
I think we need to write a better query. This is going to be executed every time it runs. We had a similar issue with the Deequ rules where it ended up either failing or creating more rules with the same value.

Can you check you can re-run the same `INSERT` twice without failing?

Reply:
aha with every time you mean with upgrades?

Reply:
changed it to replace into